### ISSUE TITLE
Replace the single shotgun model with a weak-looking pistol

### ISSUE THREAD

**Issue by squeek502** [fortressforever/fortressforever#149]
From http://forums.fortress-forever.com/showthread.php?t=25037

> Also, that "it's a shotgun" concept is actually the exact reason the single shotgun was such a problem. Shotguns generally are assumed to be good damage dealing weapons by most people. The fact that the single shotgun is actually worse than the worst pistols in the majority of games is really confusing, to the point that new players will still use it even when they are getting zero kills with it. See: https://www.youtube.com/watch?v=b8r5gRpsZLI
> 
> This might lead you to believe that it's a visual problem, not a gameplay problem, and you'd probably be right. But removing it would still be the best option until we can get a weapon model that fits the utter lack-of-damage it does. 

Some pathetic looking pistol would be a real benefit to help clarify how useless it really is and hopefully signal to new players that it shouldn't be used as a primary weapon.

**Comment by WiFiDi** [comment]
i think leaving some guide lines or developing what you want it to look like or pictures to help the artist get started might help for instance id suggest something like this.

[long range pistol](http://picturearchive.gunauction.com/4181063327/7603984/b76b027e6e6ec5206e0a24e64d88dbec.jpg)

i think something similar to that pistol would look really good along side what we have currently. ofc i think before you start on that have to establish what its fire rate and other stuff would be do you want it to be a spam fire pistol or are we just making a pistol shotgun? my concern is that if you just ask for any model is if you could end up with the same issue you had before of creating a pistol that doesn't intuitively display the feel of the weapon your intending on making.

**Comment by squeek502** [comment]
Yeah, I didn't explain it too well. It should behave exactly the same as the current single shotgun, just look different.

So, it needs to do a few things:
- [ ] Look weak
- [ ] Reload single bullets at a time

That could mean a revolver design (but would need to look different from a standard revolver [revolvers are assumed to be powerful], and be much faster than revolvers in most games), or it needs to have a magic magazine-less design where you can just load bullets one-at-a-time somewhere (where the magazine slot usually is?) and it just works.

It also doesn't need to be a standard pistol; that just follows the standard convention of shotgun > pistol that most games use. It could be a totally made up thing, or even [a superpowered air gun](http://www.airgundepot.com/air-pistols.html) or something.

**Comment by R00Ki3** [comment]
"Scatter revolver"

![thunder5](https://cloud.githubusercontent.com/assets/3948413/6887898/e84519a6-d632-11e4-8e86-9bf8c7a172ad.jpg)

An air powered gun might be the way to go to show "how weak" the gun actually is along with a weak air sound .

**Comment by Xylemon** [comment]
Now I may have played all the Fortress games for many years, so I suppose I can't speak for noobies, but hear me out on this. The single shotgun in FF does not strike me as this confusing weapon that should be more powerful. The single shotty doesn't sound or visually look powerful, so while it may be slightly odd at first that it's weak, it certainly wouldn't be confusing past the first minute or two. This should be even furthered by the fact there is another shotgun in the game which is very clearly more powerful by the model and sounds.

Another thing I feel I have to say is the above examples you guys have given above actually look more powerful than the single shotty, at least in my opinion and would be horrible replacements. Just my two cents.

**Comment by FDA-** [comment]
Xylemon we've experienced numerous examples of new players making the exact mistake outlined here. I know it seems obvious to you, but it apparently is not to new players.

**Comment by WiFiDi** [comment]
what if instead of changing the model we changed the sound and renamed it the "pellet gun" would give it a weaker feel and ofc the model could change also but after these changes are made i bet it would alieviate alot of the issues.

we could also add a co2 tank to the mdoel for emphasis
![example](http://www.everydaynodaysoff.com/wp-content/uploads/2011/01/Belt-Fed-22-Pellet-Air-Gun.jpg)

**Comment by squeek502** [comment]
That's true, maybe a simple change in sound effect/name would be enough.

@BullyFF, want to try cooking up a pellet gun sound? Something like this would probably work: https://www.sounddogs.com/sound-effects/58/mp3/549071_SOUNDDOGS__pe.mp3

More here: https://www.sounddogs.com/results.asp?Type=1&CategoryID=1027&SubcategoryID=73

**Comment by R00Ki3** [comment]
Great list of sounds on Squeek's link. But perhaps something more 'air' sounding? 
http://www.audioblocks.com/stock-audio/air-cannon-shot.html

I think a sound change and re-name would go a long way. Maybe add a co2 can to the side of the shotgun and some pipes kinda like the tranq gun.

**Comment by squeek502** [comment]
From [KevinL on the FF forums](http://forums.fortress-forever.com/showpost.php?p=507826&postcount=21):

> As a guy who just started playing the game but played some TFC before, I'd also like the Pyro to have its shotgun back. To solve the problem of players misuing it as soon as possible, I think that it could use the Half-Life 2 pistol sound that comes in the files installed with the game (specifically pistol_fire2.wav) that always sounded pathetically weak and that the gun should be renamed the "Pellet Gun" on HUD elements with a possibly hint coming up explaining its general weakness and greater effectiveness at range when selected.

That HL2 pistol sound could indeed work. It sounds remarkably weak. Even better if you up the pitch a bit.

If anyone wants to try it, edit scripts/game_sounds_ff_weapons.txt and change the `shotgun.single_shot` entry to:

```
    "shotgun.single_shot"
    {
        "channel"       "CHAN_WEAPON"
        "volume"        "1.0"
        "CompatibilityAttenuation"  "0.6"
        "pitch"         "130,140"
        "wave"          "weapons/pistol/pistol_fire2.wav"
    }
```

**Comment by BullyFF** [comment]
@squeek502 I'm free from work next week, i'll have a play around and get some samples up asap. Can you please label as sound too. Edit - Updated my Soundcloud with some attempts

**Comment by Largos** [comment]
I like airpellet 3 the best.  Good stuff.

**Comment by L1pE** [comment]
How about something that looks like a Volcanic Pistol: http://cowansauctions.com/itemImages/x0548.jpg
Only it's loaded with shorter .410-looking shells, and loaded from the behind the barrel instead of the front so it can be reloaded faster and resemble a shotgun more. The metal parts could also be made chrome and the grip be a light wood, so it resembles the old single barrel shotty from TFC a bit in its texture.

**Comment by arenaconspiracy** [comment]
judging by how it's been 7 years since the last activity on this issue, and the fact that it's called the pellet gun in game, i propose that this issue be closed, no? i'm confused on why this issue is still open... 
i'm quite literally a nobody here, and i doubt anyone looks at the issues anymore, but i'm just digging around on my own time and just felt like pointing something out. sorry.
